lib/routes/routes.js

```diff
@@ -0,0 +1,50 @@
+const { Router } = require('express');
+
+const quotes = [];
+
+module.exports = Router()
+  .post('/', (req, res) => {
+    const {
+      name,
+      favoriteCharacter,
+      quote
+    } = req.body;
```

Comment:
If you console log re.body there is not a property called quote on the body. I dont believe your doing what you think are you are doing. Currently, you do not get back a random quote.

Reply:
This is fixed, yeah?